stdlib/2/os/path.pyi

```diff
@@ -5,6 +5,7 @@
 import os
 import sys
 from typing import overload, List, Any, AnyStr, Sequence, Tuple, TypeVar, Union, Text, Callable, Optional
+from genericpath import exists
```

Comment:
Imports are not automatically re-exported from stubs. Use `from genericpath import exists as exists`.

Reply:
Thanks, I didn't know that. Made the changes.